YoCode/PathManager.cs

```diff
@@ -11,14 +11,12 @@ public class PathManager : IPathManager
         const string CS = "*.cs";
         const string SLN = "*.sln";
 
-        public IEnumerable<string> OriginalPaths { get; }
-        public IEnumerable<string> ModifiedPaths { get; }
         public IEnumerable<String> OriginalPaths { get; }
         public IEnumerable<String> ModifiedPaths { get; }
```

Comment:
👍 for fixing the duplicate properties here. 

In C#, it's more common to use the keyword `string` rather than the class name `String`, so keeping that one would have been more fitting for C#.